src/modules/Elsa.Workflows.Core/Middleware/Activities/ExceptionHandlingMiddleware.cs

```diff
@@ -60,6 +62,8 @@ public async ValueTask InvokeAsync(ActivityExecutionContext context)
 
             var strategy = await _incidentStrategyResolver.ResolveStrategyAsync(context);
             strategy.HandleIncident(context);
+
+            await context.SendSignalAsync(new ExceptionSignal(activity, e));
```

Comment:
I'm questioning whether we should report an incident if an exception was handled. What are your thoughts on that?

Reply:
This is a good question. I think if the user chooses Continue With Incidents Strategy, does he still need to add Catch Activity.If Catch Activity is not added, there will be no impact. If it is added, it means he needs this report.